sesman/chansrv/clipboard_file.c

```diff
@@ -445,6 +445,10 @@ clipboard_send_file_data(int streamId, int lindex,
     rv = send_channel_data(g_cliprdr_chan_id, s->data, size);
     free_stream(s);
     g_file_close(fd);
+
+    /* Log who transferred which file via clipboard for the purpose of audit */
+    LOG(LOG_LEVEL_INFO, "Transfered a file: filename=%s, uid=%d", full_fn, g_getuid());
```

Comment:
Rather than just saying the file is transferred, is it better to mention the direction as well, so there's no ambiguity in which way the transfer went. For Example:-

```c
"uid=%d downloaded file %s via clipboard", g_getuid(), full_fn
```

Reply:
Indeed.